pkg/csi_driver/node.go

```diff
@@ -450,3 +477,122 @@ func getFSStat(path string) (available, capacity, used, inodesFree, inodes, inod
 	inodesUsed = inodes - inodesFree
 	return
 }
+
+// nodeStageVolumeUpdateLockInfo updates lock info after NodeStageVolume succeed.
+func (s *nodeServer) nodeStageVolumeUpdateLockInfo(ctx context.Context, req *csi.NodeStageVolumeRequest) error {
+	volumeID := req.GetVolumeId()
```

Comment:
this can go below the feature gate check

Reply:
done